aries-site/src/examples/components/layer/LayerCenterExample.js

```diff
@@ -18,27 +18,36 @@ export const LayerCenterExample = () => {
             fill="vertical"
             overflow="auto"
             width={size !== 'small' ? 'medium' : undefined}
+            pad="medium"
           >
-            <Box
-              flex="grow"
-              direction="row"
-              gap="medium"
-              pad="medium"
-              height="small"
-            >
-              <Box pad={{ top: 'xsmall' }}>
-                <MailOption />
-              </Box>
-              <Box gap="medium">
-                <Heading margin="none" size="small">
+            <Box justify="between" direction="row">
+              <Box flex={false} gap="small" direction="row">
+                <Box justify="center">
+                  <MailOption />
+                </Box>
+                <Text margin="none" size="xlarge">
                   Modal Dialog
-                </Heading>
-                <Text>
-                  For modal dialogs, the use case will determine the design and
-                  size of the box for your content
                 </Text>
               </Box>
+              <Box justify="center">
+                <Button
+                  icon={<Close size="small" align="center" />}
```

Comment:
The icon doesn't look aligned when you hover over it. Do you notice that? Seems like `FormClose` icon works better and aligns with what we're doing in Filtering.

Reply:
okay let me change that and see I did notice the alignment was off without the align which didn't feel correct